tasks/main.yml

```diff
@@ -27,10 +27,10 @@
     - ctdb
 
 # Setup Geo-Replication
-- name: Setup Geo-Replication
-  import_role:
-    name: "gluster.features/roles/geo_replication"
-  when: >
-    gluster_features_georep_mastervol is defined
-  tags:
-    - georeplication
+# - name: Setup Geo-Replication
```

Comment:
@sac Are these lines expected to be commented out?
 Are we not calling the georep role?